documentation/proc-pages/physics-models/fusion_reactions/beam_reactions.md

```diff
@@ -0,0 +1,392 @@
+    $$
+
+    We can thus also define the total hot ion depoisted energy as:
+
+    $$
+    E_{\text{hot,total}} = \frac{\left( E_{\text{hot,D}}  \langle n_{\text{beam}} \rangle_{\text{D}}\right) + \left( E_{\text{hot,T}}  \langle n_{\text{beam}} \rangle_{\text{T}}\right)}{\langle n_{\text{beam}} \rangle_{\text{total}}}
+    $$
+
+7. **Calculate the hot ion species fusion rates**
+
+    The D-T fusion reaction rate is calculated from the [`beam_reaction_rate()`](#beam-fusion-reaction-rate--beam_reaction_rate) function.
+
+    ------------------------
+
+    ### Beam fusion reaction rate | `beam_reaction_rate()`
+
+    1. **Calculate beam velocity**
+
+        The beam velocity ($v_{\text{beam}}$) is calculated from the inputted beam energy and relative ion mass of the beam by simply re-arranging the kinetic energy equation
+
+    2. **Define the integral coefficient**
+
+        $$
+        \frac{3v_{\text{critical}}}{\ln\left(1+\frac{v_{\text{beam}}}{v_{\text{critical}}}\right)^{3}}
+        $$
+
+    3. **Perform the fusion rate integral**
+
+        $$
+        \int_0^{v_{\text{relative}}} \frac{u^3}{1+u^3}\sigma_{\text{bmfus}}(E_{\text{amu}})
+        $$
+        
+        --------------------
+
+        #### Hot Beam Fusion Reaction Rate Integrand | `_hot_beam_fusion_reaction_rate_integrand()`
+
+        This function computes the integrand for the hot beam fusion reaction rate based on the ratio of beam velocity to the critical velocity and the critical velocity for electron/ion slowing down of the beam ion.
+
+        The integrand function is:
+
+        $$
+        \int \frac{u^3}{1+u^3}\sigma_{\text{bmfus}}(E_{\text{amu}})
+        $$
+
+        Where $u$ is the inputted ratio of the beam to the critical velocity.
+        $E_{\text{amu}}$ represents the beam kinetic energy per atomic mass unit.
+
+        The calculated beam fusion cross section $\sigma_{\text{bmfus}}$ is calculated from [`_beam_fusion_cross_section()`](#beam-fusion-cross-section--_beam_fusion_cross_section)
+
+        ------------------------
+
+        #### Beam fusion cross section | `_beam_fusion_cross_section()`
+
+        This internal function is used to find the beam cross section.
+        It sets limits on cross-section at low and high beam energies. The plasma ions are assumed to be stationary:
+
+        $$
+        \sigma_{\text{bm}}(E) = 
+        \begin{cases} 
+        1.0 \times 10^{-27} \ \text{cm}^2 & \text{if } E < 10.0 \ \text{keV/amu} \\
+        8.0 \times 10^{-26} \ \text{cm}^2 & \text{if } E > 10^4 \ \text{keV/amu} \\
+        \frac{1.0 \times 10^{-24} \cdot \left( \frac{a_2}{1.0 + (a_3 E - a_4)^2} + a_5 \right)}{E \left( \exp\left(\frac{a_1}{\sqrt{E}}\right) - 1.0 \right)} \ \text{cm}^2 & \text{otherwise}
+        \end{cases}
+        $$
+
+        where:
+
+        - \( E \) is the beam energy in, $\text{keV/amu}$
+
+        - \( a_1, a_2, a_3, a_4, a_5 \) are constants.
```

Comment:
source for these constants and values?

Reply:
I have looked far and wide and cannot find the source for this.